src/DatadogTestLogger/DatadogTestResultSerializer.cs

```diff
@@ -59,19 +60,52 @@ public string Serialize(LoggerConfiguration loggerConfiguration, TestRunConfigur
 
     private void ShowEnvironmentVariables(StringBuilder builder, string title)
     {
+        const string defaultRegexPattern = @"^(DD_|COR_|CORECLR_|BUILD_|SYSTEM_).*";
```

Comment:
I still get a bit nervous seeing the `BUILD_` and `SYSTEM_` regex, as we can't know if they're safe or not... IMO we shouldn't be logging anything we don't _know_ is safe, i.e. DD_ or COR_ variables. Otherwise I think we should have a static, fixed list of variables. Not a regex/glob.

Reply:
ok let's change the default to be safe.

Reply:
done in [17c6466](https://github.com/tonyredondo/datadog-test-logger/pull/19/commits/17c64661267ced037ce3e224e8332375efcc6fe9)